ShopBonsai/no-hassle
# 💥⬆️ BREAKING - Upgrade Joi 15 to 17 + OpenAPI 3

## What changes are made

- Upgrade Joi from 15 to 17
- Generate OpenAPI3 instead of OpenAPI2 code

## How to Test

- Let Continuous Integration do its work
- After publishing `.rc` package, use it within an existing project to see what needs to change as this package unfortunately doesn't have enough automated tests
